gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -964,6 +964,88 @@ public static DiskFilter notEquals(DiskField field, long value) {
   }
 
   /**
+   * Class for filtering subnetwork lists.
+   */
+  class SubnetworkFilter extends ListFilter {
+
+    private static final long serialVersionUID = 979448583739105481L;
+
+    private SubnetworkFilter(SubnetworkField field, ComparisonOperator operator, Object value) {
+      super(field.selector(), operator, value);
+    }
+
+    /**
+     * Returns an equals filter for the given field and string value. For string fields,
+     * {@code value} is interpreted as a regular expression using RE2 syntax. {@code value} must
+     * match the entire field.
+     *
+     * @see <a href="https://github.com/google/re2/wiki/Syntax">RE2</a>
+     */
+    public static SubnetworkFilter equals(SubnetworkField field, String value) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method name and parameter number are suspiciously close to equals(Object)](https://www.codacy.com/app/mziccard/gcloud-java/file/2431752990/issues/source?bid=3140968&fileBranchId=3241696#l984)